components/needs_selector.js

```diff
@@ -23,12 +37,23 @@ const styles = theme => ({
     color: "white"
   },
   clearButton: {
-    textAlign: "right",
     textDecoration: "underline"
+  },
+  gridItemButton: {
+    textAlign: "center"
   }
 });
 
 class NeedsSelector extends Component {
+  state = {
+    open: true
```

Comment:
Should the default state be closed on mobile? Probably worth checking with Raluca or Michelle.

Reply:
changed default mobile state to closed based on discussion w/ Raluca

Reply:
(might be good to commit those changes 😛 )